### PR TITLE
Old style exceptions --> new style for Python 3

### PR DESCRIPTION
**Pull request by cclauss**
## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue
Syntax modifications to be compatible with both Python2 and Python 3.

## Description
A few sentences describing the overall goals of the pull request's commits.
```
git checkout -b modernize-python2-code
python3 -m flake8 . --count --show-source --statistics --select=E9,F63,F72,F82
futurize -f lib2to3.fixes.fix_except <files old style exceptions that flake8 found>
futurize -f libfuturize.fixes.fix_print_with_import <file with legacy printing that flake8 found>
```

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 4.5.0
- [ ] 5.0.0
- [ ] 5.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests -- flake8 command above
- [ ] Documentation 

## Demisto Partner?
- [ ] The title must be in the following format: **[YOUR_PARTNER_ID] short description**

### CODE REVIEW

File: Packs/IBMResilientSystems/Integrations/IBMResilientSystems/IBMResilientSystems.py [hunk 1]
```diff
@@ -1038,7 +1038,7 @@ def test():
     elif demisto.command() == 'rs-related-incidents':
         demisto.results(related_incidents_command(demisto.args()['incident-id']))
 
-except Exception, e:
+except Exception as e:
```
Comment (ShahafBenYakir) [inline Packs/IBMResilientSystems/Integrations/IBMResilientSystems/IBMResilientSystems.py:1041]: This is python 2 integration, not python 3

File: Packs/DeprecatedContent/Integrations/Cymon/Cymon.py [hunk 2]
```diff
@@ -460,5 +460,5 @@ def http_request(method, url, headers):
     elif command == 'domain':
         cymon_says()
 
-except Exception, e:
+except Exception as e:
```
Comment (ShahafBenYakir) [inline Packs/DeprecatedContent/Integrations/Cymon/Cymon.py:463]: This is python 2 integration, not python 3

File: Packs/BigFix/Integrations/BigFix/BigFix.py [hunk 3]
```diff
@@ -791,7 +791,7 @@ def test():
     elif demisto.command():
         query_command()
 
-except Exception, e:
+except Exception as e:
```
Comment (ShahafBenYakir) [inline Packs/BigFix/Integrations/BigFix/BigFix.py:794]: This is python 2 integration, not python 3

File: Packs/Ipstack/Integrations/Ipstack/Ipstack.py [hunk 4]
```diff
@@ -100,5 +100,5 @@ def test_module():
         test_module()
     elif demisto.command() == 'ip':
         do_ip_command()
-except Exception, e:
+except Exception as e:
```
Comment (ShahafBenYakir) [inline Packs/Ipstack/Integrations/Ipstack/Ipstack.py:103]: This is python 2 integration, not python 3

File: Packs/McAfee_Advanced_Threat_Defense/Integrations/McAfee_Advanced_Threat_Defense/McAfee_Advanced_Threat_Defense.py [hunk 5]
```diff
@@ -779,7 +779,7 @@ def main():
         # return detonate(3, args.url, args.timeout, args.format, args.threshold);
         # submit type for url-download is 3
 
-    except Exception, ex:
+    except Exception as ex:
```
Comment (ShahafBenYakir) [inline Packs/McAfee_Advanced_Threat_Defense/Integrations/McAfee_Advanced_Threat_Defense/McAfee_Advanced_Threat_Defense.py:782]: This is python 2 integration, not python 3

File: Packs/ProofpointServerProtection/Integrations/ProofpointServerProtection/ProofpointServerProtection.py [hunk 6]
```diff
@@ -49,7 +49,7 @@ def http_request(method, url_suffix, headers=None, data=None, allow_redirects=Tr
         )
         if res.status_code not in {200, 302}:
             raise Exception('Your request failed with the following error: ' + res.content + str(res.status_code))
-    except Exception, e:
+    except Exception as e:
```
Comment (ShahafBenYakir) [inline Packs/ProofpointServerProtection/Integrations/ProofpointServerProtection/ProofpointServerProtection.py:52]: This is python 2 integration, not python 3

File: Packs/RTIR/Integrations/RTIR/RTIR.py [hunk 7]
```diff
@@ -960,7 +960,7 @@ def main():
         elif demisto.command() == 'rtir-add-reply':
             add_reply()
 
-    except Exception, e:
+    except Exception as e:
```
Comment (ShahafBenYakir) [inline Packs/RTIR/Integrations/RTIR/RTIR.py:963]: This is python 2 integration, not python 3

File: Packs/Recorded_Future/Integrations/Recorded_Future/Recorded_Future.py [hunk 8]
```diff
@@ -58,7 +58,7 @@ def http_request(method, url_suffix, params=None):
             else:
                 error_str = 'Request failed, status: ' + str(res.status_code) + ', details: ' + res.text
             return_error(error_str)
-    except Exception, e:
+    except Exception as e:
```
Comment (ShahafBenYakir) [inline Packs/Recorded_Future/Integrations/Recorded_Future/Recorded_Future.py:61]: This is python 2 integration, not python 3

File: Packs/Recorded_Future/Integrations/Recorded_Future/Recorded_Future.py [hunk 9]
```diff
@@ -1756,7 +1756,7 @@ def fetch_incidents():
     elif demisto.command() == 'recorded-future-get-alerts':
         get_alerts_command()
 
-except Exception, e:
+except Exception as e:
```
Comment (ShahafBenYakir) [inline Packs/Recorded_Future/Integrations/Recorded_Future/Recorded_Future.py:1759]: This is python 2 integration, not python 3

File: Packs/Stealthwatch_Cloud/Integrations/Stealthwatch_Cloud/Stealthwatch_Cloud.py [hunk 10]
```diff
@@ -55,7 +55,7 @@ def http_request(method, url_suffix, params_dict=None, headers=DEFAULT_HEADERS,
         except ValueError:
             # in case the response doesn't have JSON
             return "Request completed"
-    except Exception, e:
+    except Exception as e:
```
Comment (ShahafBenYakir) [inline Packs/Stealthwatch_Cloud/Integrations/Stealthwatch_Cloud/Stealthwatch_Cloud.py:58]: This is python 2 integration, not python 3

File: Packs/Stealthwatch_Cloud/Integrations/Stealthwatch_Cloud/Stealthwatch_Cloud.py [hunk 11]
```diff
@@ -558,7 +558,7 @@ def fetch_incidents():
         demisto.results(list_sessions_command())
     elif demisto.command() == 'fetch-incidents':
         demisto.results(fetch_incidents())
-except Exception, e:
+except Exception as e:
```
Comment (ShahafBenYakir) [inline Packs/Stealthwatch_Cloud/Integrations/Stealthwatch_Cloud/Stealthwatch_Cloud.py:561]: This is python 2 integration, not python 3

File: Packs/SymantecEndpointProtection/Integrations/SymantecEndpointProtection_V2/SymantecEndpointProtection_V2.py [hunk 12]
```diff
@@ -155,7 +155,7 @@ def parse_response(resp):
             return resp
         try:
             return resp.json()
-        except Exception, ex:
+        except Exception as ex:
```
Comment (ShahafBenYakir) [inline Packs/SymantecEndpointProtection/Integrations/SymantecEndpointProtection_V2/SymantecEndpointProtection_V2.py:158]: This is python 2 integration, not python 3

File: Packs/SymantecEndpointProtection/Integrations/SymantecEndpointProtection_V2/SymantecEndpointProtection_V2.py [hunk 13]
```diff
@@ -836,5 +836,5 @@ def move_client_to_group_command(token):
         move_client_to_group_command(token)
     if current_command == 'sep-identify-old-clients':
         old_clients_command(token)
-except Exception, ex:
+except Exception as ex:
```
Comment (ShahafBenYakir) [inline Packs/SymantecEndpointProtection/Integrations/SymantecEndpointProtection_V2/SymantecEndpointProtection_V2.py:839]: This is python 2 integration, not python 3

File: Packs/IBMResilientSystems/Integrations/IBMResilientSystems/IBMResilientSystems.py (hunk 1 above)
Comment (cclauss) [reply Packs/IBMResilientSystems/Integrations/IBMResilientSystems/IBMResilientSystems.py:1041]: This syntax works in BOTH.

File: Packs/DeprecatedContent/Integrations/Cymon/Cymon.py (hunk 2 above)
Comment (cclauss) [reply Packs/DeprecatedContent/Integrations/Cymon/Cymon.py:463]: This syntax works in BOTH.

File: Packs/BigFix/Integrations/BigFix/BigFix.py (hunk 3 above)
Comment (cclauss) [reply Packs/BigFix/Integrations/BigFix/BigFix.py:794]: This syntax works in BOTH.

File: Packs/Ipstack/Integrations/Ipstack/Ipstack.py (hunk 4 above)
Comment (cclauss) [reply Packs/Ipstack/Integrations/Ipstack/Ipstack.py:103]: This syntax works in BOTH.

File: Packs/McAfee_Advanced_Threat_Defense/Integrations/McAfee_Advanced_Threat_Defense/McAfee_Advanced_Threat_Defense.py (hunk 5 above)
Comment (cclauss) [reply Packs/McAfee_Advanced_Threat_Defense/Integrations/McAfee_Advanced_Threat_Defense/McAfee_Advanced_Threat_Defense.py:782]: This syntax works in BOTH.

File: Packs/Recorded_Future/Integrations/Recorded_Future/Recorded_Future.py (hunk 9 above)
Comment (cclauss) [reply Packs/Recorded_Future/Integrations/Recorded_Future/Recorded_Future.py:1759]: This syntax works in BOTH.

File: Packs/Stealthwatch_Cloud/Integrations/Stealthwatch_Cloud/Stealthwatch_Cloud.py (hunk 11 above)
Comment (cclauss) [reply Packs/Stealthwatch_Cloud/Integrations/Stealthwatch_Cloud/Stealthwatch_Cloud.py:561]: This syntax works in BOTH.

File: Packs/SymantecEndpointProtection/Integrations/SymantecEndpointProtection_V2/SymantecEndpointProtection_V2.py (hunk 12 above)
Comment (cclauss) [reply Packs/SymantecEndpointProtection/Integrations/SymantecEndpointProtection_V2/SymantecEndpointProtection_V2.py:158]: This syntax works in BOTH.

File: Packs/SymantecEndpointProtection/Integrations/SymantecEndpointProtection_V2/SymantecEndpointProtection_V2.py (hunk 13 above)
Comment (cclauss) [reply Packs/SymantecEndpointProtection/Integrations/SymantecEndpointProtection_V2/SymantecEndpointProtection_V2.py:839]: This syntax works in BOTH.

File: Packs/Stealthwatch_Cloud/Integrations/Stealthwatch_Cloud/Stealthwatch_Cloud.py (hunk 10 above)
Comment (cclauss) [reply Packs/Stealthwatch_Cloud/Integrations/Stealthwatch_Cloud/Stealthwatch_Cloud.py:58]: This syntax works in BOTH.

File: Packs/ProofpointServerProtection/Integrations/ProofpointServerProtection/ProofpointServerProtection.py (hunk 6 above)
Comment (cclauss) [reply Packs/ProofpointServerProtection/Integrations/ProofpointServerProtection/ProofpointServerProtection.py:52]: This syntax works in BOTH.

File: Packs/RTIR/Integrations/RTIR/RTIR.py (hunk 7 above)
Comment (cclauss) [reply Packs/RTIR/Integrations/RTIR/RTIR.py:963]: This syntax works in BOTH.

File: Packs/Recorded_Future/Integrations/Recorded_Future/Recorded_Future.py (hunk 8 above)
Comment (cclauss) [reply Packs/Recorded_Future/Integrations/Recorded_Future/Recorded_Future.py:61]: This syntax works in BOTH.

File: Packs/McAfee_Advanced_Threat_Defense/Integrations/McAfee_Advanced_Threat_Defense/McAfee_Advanced_Threat_Defense.py (hunk 5 above)
Comment (cclauss) [reply Packs/McAfee_Advanced_Threat_Defense/Integrations/McAfee_Advanced_Threat_Defense/McAfee_Advanced_Threat_Defense.py:782]: This syntax works in BOTH.

File: Packs/ProofpointServerProtection/Integrations/ProofpointServerProtection/ProofpointServerProtection.py (hunk 6 above)
Comment (cclauss) [reply Packs/ProofpointServerProtection/Integrations/ProofpointServerProtection/ProofpointServerProtection.py:52]: This syntax works in BOTH.

**Comment by ShahafBenYakir** [comment]
Hi @cclauss In order to merge this I will need you to sign this:
https://github.com/demisto/content/pull/8202#issuecomment-663748948

**Pull request merged**